services/storage/src/simcore_service_storage/simcore_s3_dsm_utils.py

```diff
@@ -78,6 +78,16 @@ async def expand_directory(
     return result
 
 
+async def try_get_fmd(
```

Comment:
I think this is OK.
```suggestion
async def get_fmd(
```

Reply:
@GitHK this was previously a private function that silences exception. that is ok. but making it now public and it silences exception is a bit against most of our practices everywhere else in the code.

Reply:
Trying another way, the function will return at the origins.